go/kbnm/handler.go

```diff
@@ -24,8 +25,21 @@ func handleChat(req *Request) error {
 	}
 
 	// FIXME: Get the absolute path without a filled PATH var somehow?
-	cmd := exec.Command("/usr/local/bin/keybase", "chat", "send", "--private", req.To, req.Body)
+	cmd := exec.Command("/usr/local/bin/keybase", "chat", "send", "--private", req.To)
 
-	// TODO: Check/convert status code more precisely?
+	// Write message body over STDIN to avoid running up against bugs with
+	// super long messages.
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		return err
+	}
+
+	go func() {
+		io.WriteString(stdin, req.Body)
+		stdin.Close()
+	}()
```

Comment:
Yes, the `io.WriteString(stdin, req.Body)` line blocks until it gets consumed. The stdin doesn't get consumed until `cmd.Run()` gets run below. If it isn't in a goroutine, it deadlocks.